apps/wpcom-block-editor/src/wpcom/features/add-tags-education-link.js

```diff
@@ -0,0 +1,33 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { ExternalLink } from '@wordpress/components';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { addFilter } from '@wordpress/hooks';
+import { useTranslate } from 'i18n-calypso';
+import tracksRecordEvent from './tracking/track-record-event';
+
+const trackTagsEducationLinkClick = () =>
+	tracksRecordEvent( 'wpcom_block_editor_tags_education_link_click' );
+
+const addTagsEducationLink = createHigherOrderComponent( ( PostTaxonomyType ) => {
+	return ( props ) => {
+		const translate = useTranslate();
+
+		if ( props.slug !== 'post_tag' ) {
+			return <PostTaxonomyType { ...props } />;
+		}
+
+		return (
+			<>
+				<PostTaxonomyType { ...props } />
+				<ExternalLink
+					href={ localizeUrl( 'https://wordpress.com/support/posts/tags/' ) }
```

Comment:
I'm not sure `localizeUrl()` will work in the editor context ... the package also exports a `useLocalizeUrl()` hook, and I think that's able to automatically find what the current locale is (the editor doesn't have an easy way to discover what the current locale is so we had to write a bunch of a8c specific code to figure it out)